packages/validator/test/scoring-tool/rubric.test.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright 2024 IBM Corporation.
+ * SPDX-License-Identifier: Apache2.0
+ */
+
+const path = require('path');
+const { getCategories } = require('../../src/scoring-tool/categories');
+const readYaml = require('../../src/cli-validator/utils/read-yaml.js');
+const rubric = require('../../src/scoring-tool/rubric');
+const validateSchema = require('../../src/cli-validator/utils/validate-schema');
+
+const ibmRuleset = require('@ibm-cloud/openapi-ruleset');
+
+describe('scoring-tool rubric tests', function () {
```

Comment:
These tests are essentially "linters" for the rubric, to make sure it stays up to date and uses the right data.